src/plays/tenzies-game/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Tenzies Game
+
+Tenzies Game using React.Js
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: AbhiPatel10
+- Gihub Link: https://github.com/AbhiPatel10
+- Blog:
+- Video:
+
+## Implementation Details
+
+Update your implementation idea and details here
```

Comment:
Provide the implementation details. Imagine someone will look into this and understand how it was implemented.

Reply:
Okay I will Update Readme.md file.